include/sound/sdca_function.h

```diff
@@ -775,6 +775,34 @@ struct sdca_entity_cs {
 	unsigned int max_delay;
 };
 
+/**
+ * struct sdca_pde_delay - describes the delay changing between 2 power states
+ * @from_ps: The power state being exited.
+ * @to_ps: The power state being entered.
+ * @us: The delay in microseconds switching between the two states.
+ */
+struct sdca_pde_delay {
+	int from_ps;
+	int to_ps;
+	int us;
```

Comment:
it's possible that the delay is > 64k us, I would use u32

Reply:
It's not clear if we should care about such delays, except maybe to implement a timeout when the transition to the desired power state is stuck?

Reply:
Yeah I am imagining the delays would likely be used either as delays or as timeouts in DAPM widgets for the PDEs. But either way even if we didn't no harm in them being parsed anyways. It is worth I did only parse the max delay, as I couldn't see any use for the typical delay. Both delays and timeouts would have to run on the max delay.